src/vaski-hop.js

```diff
@@ -135,24 +180,24 @@ function announceGameResult() {
 	window.alert(resultTexts[gameOutcome])
 };
 
-var displayComputerSelection = function () {
-
+var selectionNotMade = function () {
 	if (playerSelection === -1 || playerSelection === undefined) {
-		playerName.innerHTML = "Please select something first!!!";
-		playerName.classList.add('errors')
-		return
+		return true;
+	} else {
+		return false
+	}
+};
+
+var continueGame = function () {
+
+	if (selectionNotMade()) {
+		throwSelectionError();
 	} else {
+		hideContinueButton();
 		setComputerSelection();
-		continueButton.classList.add('hidden');
-
-		if (computerSelection == typeScissors) {
-			scissors2.classList.remove('hidden')
-		} else if (computerSelection == typePaper) {
-			paper2.classList.remove('hidden')
-		} else if (computerSelection == typeWell) {
-			well2.classList.remove('hidden')
-		}
-		calcWinner();
+		unhideComputerImg();
```

Comment:
o jeigu keisi žaidimo struktūrą ar eigą( logiką ar taisykles) ir continueGame iškviesi kažkuriuo metu, kai computerImg bus matomas? Tai unhideComputer bus melas - nes jis nebuvo hidden. Pasirinkdamas tokį pavadinimą tu darai sprendimą (įsipareigojimą) kviesti šitą funkciją griežtai tik tada kai tas image'as yra hidden. Kam daryti tokį įsipareigojimą?
Pvz. gal ir ne gereiausias pavadinimas bet ensureComputerImgIsShown() - nedaro prielaidos kad jis bus hidden ar shown, vadinasi atitiks bet kurią situaciją.